splice_spark/pom.xml

```diff
@@ -317,22 +317,7 @@
                     </execution>
                     <execution>
                         <id>############### stop-splice-its ###############</id>
-                        <phase>prepare-package</phase>
```

Comment:
i'm confused about this 'stop test' removal. where is it being done now, instead?

Reply:
@MurrayBrown The stop tests task is in two phases before: prepare-package and pre-clean. prepare-package is not needed before because other IT will stop the Spice database. pre-client is too late. So I change the phases into post-integration-test so that the Splice database will stop after this module's IT.